### PR TITLE
[BEAM-2241] Mark PipelineVisitor and AppliedPTransform as internal.

### PR DESCRIPTION
**Pull request by mariapython**

### CODE REVIEW

**Comment by mariapython** [comment]
R: @aaltay

File: sdks/python/apache_beam/pipeline.py [hunk 1]
```diff
@@ -390,9 +391,10 @@ def leave_composite_transform(self, transform_node):
 
 
 class AppliedPTransform(object):
-  """A transform node representing an instance of applying a PTransform.
+  """For internal use only; no backwards-compatibility guarantees.
 
-  This is an internal class used for bookkeeping by a Pipeline.
+  A transform node representing an instance of applying a PTransform
+  (used internally by Pipeline for bookeeping purposes)..
```
Comment (aaltay) [inline sdks/python/apache_beam/pipeline.py]: Remove the extra `.` at the end of the sentence.

**Comment by aaltay** [comment]
LGTM except for the one comment above

@chamikaramj could you coordinate merging and cherry-picking of changes related to BEAM-2241. (I can review any PRs.)

**Comment by chamikaramj** [comment]
Sounds good. I'll merge.

**Pull request closed without merging**